modules/authentication/proc-enabling-user-authentication-with-rhbk.adoc

```diff
@@ -25,11 +25,158 @@ Save the value for the next step:
++
+`AUTH_OIDC_CLIENT_ID`::
+Enter the saved **Client ID**.
+
+`AUTH_OIDC_CLIENT_SECRET`::
+Enter the saved **Client Secret**.
+
+`AUTH_OIDC_METADATA_URL`::
+Enter the saved **{rhbk} realm base URL**.
+
+. Enable the Keycloak organization plugin (`backstage-plugin-catalog-backend-module-keycloak-dynamic`).
+The plugin is named after {rhbk} upstream project.
+This plugin ingests {rhbk} users and groups to the {product-short} software catalog.
++
+.`dynamic-plugins.yaml` file fragment
+[source,yaml]
+----
+plugins:
+  - package: './dynamic-plugins/dist/backstage-plugin-catalog-backend-module-keycloak-dynamic'
+    disabled: false
+----
+
+. To provision {rhbk} users and groups to the {product-short} software catalog, add the `catalog.providers.keycloakOrg` section to your custom {product-short} `{my-app-config-file}` configuration file:
++
+--
+[id=keycloakOrgProviderId]
+.`{my-app-config-file}` fragment with mandatory `keycloakOrg` fields
+[source,yaml]
+----
+catalog:
+  providers:
+    keycloakOrg:
+      default:
+        baseUrl: ${AUTH_OIDC_METADATA_URL}
+        clientId: ${AUTH_OIDC_CLIENT_ID}
+        clientSecret: ${AUTH_OIDC_CLIENT_SECRET}
+----
+
+`baseUrl`::
+Your {rhbk} server URL, defined when xref:enabling-user-authentication-with-rhbk[enabling authentication with {rhbk}].
+
+`clientId`::
+Your {product-short} application client ID in {rhbk}, defined when xref:enabling-user-authentication-with-rhbk[enabling authentication with {rhbk}].
+
+`clientSecret`::
+Your {product-short} application client secret in {rhbk}, defined when xref:enabling-user-authentication-with-rhbk[enabling authentication with {rhbk}].
+
+Optional: Consider adding the following optional fields:
+
+`realm`::
+Realm to synchronize.
+Default value: `master`.
 +
-`AUTH_OIDC_CLIENT_ID`:: Enter the saved **Client ID**.
-`AUTH_OIDC_CLIENT_SECRET`:: Enter the saved **Client Secret**.
-`AUTH_OIDC_METADATA_URL`:: Enter the saved **{rhbk} realm base URL**.
+.`{my-app-config-file}` fragment with optional `realm` field
+[source,yaml]
+----
+catalog:
+  providers:
+    keycloakOrg:
+      default:
+        realm: master
+----
+
+`loginRealm`::
+Realm used to authenticate.
+Default value: `master`.
++
+.`{my-app-config-file}` fragment with optional `loginRealm` field
+[source,yaml]
+----
+catalog:
+  providers:
+    keycloakOrg:
+      default:
+        loginRealm: master
+----
+
+`userQuerySize`::
+User number to query simultaneously.
+Default value: `100`.
++
+.`{my-app-config-file}` fragment with optional `userQuerySize` field
+[source,yaml]
+----
+catalog:
+  providers:
+    keycloakOrg:
+      default:
+        userQuerySize: 100
+----
+
+`groupQuerySize`::
+Group number to query simultaneously.
+Default value: `100`.
++
+.`{my-app-config-file}` fragment with optional `groupQuerySize` field
+[source,yaml]
+----
+catalog:
+  providers:
+    keycloakOrg:
+      default:
+        groupQuerySize: 100
+----
+
+`schedule.frequency`::
```

Comment:
@JessicaJHee Is schedule an optional section in the case of RHBK / Keycloak? Or is it a mandatory field like for GitHub?

Reply:
Answer from @JessicaJHee : yes, it's optional.